app/services/widget_searcher.rb

```diff
@@ -0,0 +1,64 @@
+class WidgetSearcher
+
+  #1. assign @queries to all search options in the params (these should be populated from the filter form)
```

Comment:
Just wanted to flag that this is a really high quality comment block. Nice job. :smile: